05_timers/time_manager.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * COPYRIGHT (C) 2021 Oleksandr Aleksieiev
+ * all rights reserved
+*/
+#include <iomanip>
+#include "time_manager.h"
+
+namespace time_manager {
+    const std::map<const clockid_t, std::string> TimeManager::validClockIdsMap = {
```

Comment:
Why not const std::string&

Reply:
What for?

Reply:
Pair will pass a const reference, this is already solved by STL